arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/transients/DeploymentInfo.java

```diff
@@ -61,6 +60,11 @@ private int getDeploymentNumber(Address address) {
     return this.deploymentNumber.getOrDefault(address, 0);
   }
 
+  public void deploy(Address address) {
```

Comment:
This already existed, didn't it ? I had this method on my branch, too. I'm confused.

Reply:
It was part of the conflicts I solved during a merge of `arith-dev `into this branch. Probably there were two versions and I kept the one of `arith-dev`.